llvm/test/Transforms/LoopVectorize/AArch64/struct-return.ll

```diff
@@ -0,0 +1,284 @@
+; RUN: opt < %s -passes=loop-vectorize,dce,instcombine -force-vector-interleave=1 -S | FileCheck %s --check-prefixes=NEON
```

Comment:
> Does this test need to be specific to SVE? 

Not really, but I find convincing the vectorizer to use scalable vectors without `+sve` quite finicky... I was playing with `-force-target-supports-scalable-vectors=true`, but it seems not to try scalable VFs unless I force the vector width too, and then it's hit or miss. I'm probably doing something wrong, so I'll play around with it a bit more.

Reply:
The target independent tests don't have to necessarily use scalable vectors, it would be sufficient to cover the various code paths for fixed vectors. And then have some scalable codegen & AArch64 cost modeling tests in `AArch64/`

Reply:
Alright, I've split the fixed-size tests into a target agnostic test, and left the scalable tests under AArch64 :+1: 
Also, added another test to print the VPlan (when widening or replicating struct returns).